adminapi/base.py

```diff
@@ -27,16 +27,9 @@ def __bool__(self):
         return bool(self.get_results())
 
     def __repr__(self):
-        # QuerySet is not used directly but through query function
-        kwargs = ', '.join(
-            '{0}={1!r}'.format(k, v) for k, v in self._filters.items()
-        )
-        query_repr = 'query({0})'.format(kwargs)
-
-        if self._restrict:
-            query_repr += '.restrict({0})'.format(', '.join(self._restrict))
-
-        return query_repr
+        return 'Query({{{}}})'.format(', '.join(
```

Comment:
`{{{}}}` seriously? 3 nested format blocks? I even didn't try to understand that. ^^ What does it do? Why 3 nested blocks?

Reply:
I added a comment.